tests/dummy_qrc/qm.py

```diff
@@ -13,37 +13,27 @@ def create(runcard=RUNCARD):
 
     Used in ``test_instruments_qm.py`` and ``test_instruments_qmsim.py``
     """
-    # Create channel objects
-    channels = ChannelMap()
-    # readout
-    channels |= ("L3-25_a", "L3-25_b")
-    # feedback
-    channels |= ("L2-5_a", "L2-5_b")
-    # drive
-    channels |= (f"L3-{i}" for i in range(11, 16))
-    # flux
-    channels |= (f"L4-{i}" for i in range(1, 6))
-    # TWPA
-    channels |= "L4-26"
+    controller = QMSim("qmopx", "0.0.0.0:0", simulation_duration=1000, cloud=False)
+    # set time of flight for readout integration (HARDCODED)
+    controller.time_of_flight = 280
```

Comment:
Actually, why this can't be set in the constructor, together with the other params?

Reply:
Fixed, thanks.